lib/private/Files/Template/TemplateManager.php

```diff
@@ -175,7 +175,11 @@ public function createFromTemplate(string $filePath, string $templateId = '', st
 	 */
 	private function getTemplateFolder(): Node {
```

Comment:
```suggestion
	private function getTemplateFolder(): Folder {
```

Just for a tiny bit more safety ;)